CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## Unreleased
 
+## v5.0.0 - Unreleased
+
+- [Breaking] Range types are now (mostly) immutable [#142](https://github.com/octoenergy/xocto/pull/142)
+    - start, end, and bounds can no longer be modified after creation
+- [Breaking] The `types` module is now split into a package containing `types.generic` and `types.djagno` [#144](https://github.com/octoenergy/xocto/pull/144)
```

Comment:
```suggestion
- [Breaking] The `types` module is now split into a package containing `types.generic` and `types.django` [#144](https://github.com/octoenergy/xocto/pull/144)
```

Reply:
I make this spelling mistake **all the time**